analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisablingRequestValidation.vb

```diff
@@ -0,0 +1,109 @@
+﻿Imports System
```

Comment:
_Rebase conflict:_ This file should not be here. It's already on master and was moved to `TestCases/Hotspots`

Reply:
thanks for addressing them in d633c0787f5